### PR TITLE
Make ARM Audio max volume configurable

### PR DESCRIPTION
**Pull request by drashna**

### CODE REVIEW

**Comment by drashna** [comment]
Hmmm, maybe this doesn't work how I think.

Closing it for now.

File: docs/feature_audio.md [hunk 1]
```diff
@@ -65,6 +65,13 @@ The available keycodes for audio are:
 * `AU_OFF` - Turn audio mode off
 * `AU_TOG` - Toggle audio mode
 
+## Arm Audio Volume
```
Comment (mechmerlin) [inline docs/feature_audio.md]:
```suggestion
## ARM Audio Volume
```

File: docs/feature_audio.md [hunk 2]
```diff
@@ -65,6 +65,13 @@ The available keycodes for audio are:
...
+
+For ARM devices, you can adjust the DAC Sample values. If your board is too loud for you or your coworkers, you can set the max using `DAC_SAMPLE_MAX` in your `config.h`:
```
Comment (mechmerlin) [inline docs/feature_audio.md]:
```suggestion
For ARM devices, you can adjust the DAC sample values. If your board is too loud for you or your coworkers, you can set the max using `DAC_SAMPLE_MAX` in your `config.h`:
```

**Comment by drashna** [comment]
>Sounds good to me.

..... ROFLMAO

**Pull request merged**